packages/client-core/src/systems/WidgetUISystem.ts

```diff
@@ -202,6 +209,13 @@ const execute = () => {
 }
 
 const reactor = () => {
+  const xrState = useHookstate(getMutableState(XRState))
+  useEffect(() => {
+    if (!xrState.sessionActive.value) {
+      const { widgetMenuUI } = getState(WidgetUISystemState)
+      removeComponent(widgetMenuUI.entity, VisibleComponent)
```

Comment:
Let's use the showWidget action to hide all widgets instead of just removing the visible component

Reply:
You know I attempted to do so but it wasn't working. I didn't realize until now that the execute function was doing the removal and I had disabled execute with an XrState check while working on this. Let me revert to using the showWidgetQueue.